Whisper_Audio.ipynb

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
1. I suggest setting all variables in a single section:
audio_folder, transcription_folder, lang
2. You can change the names to audio_folder_path and transcription_folder_path.
3. You can add another variable for file suffix (.txt)